src/components/Cart/Cart.js

```diff
@@ -1,49 +1,133 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
-import CartProductsListItem from '../CartProductsListItem/CartProductsListItem';
+import CartList from '../CartProductsListItem/CartProductsListItem';
 import './Cart.scss';
 
-const Cart = ({ isCartOpen, setIsCartOpen }) => {
+const Cart = ({ setIsCartOpen }) => {
+  const [cartData, setCartData] = useState([]);
   const navigate = useNavigate();
 
+  const clickOrder = () => {
+    navigate('/order');
+    setIsCartOpen(false);
+  };
+
+  useEffect(() => {
+    fetch('http://10.58.52.228:8002/carts')
+      .then(response => response.json())
+      .then(data => setCartData(data.carts));
+  }, []);
+
+  const handleItemNum = (value, quantity, cartId) => {
+    if (quantity + value === 0) return;
+
+    const newArr = [...cartData];
+
+    setCartData(
+      newArr.map(list => {
+        if (list.cartId === cartId) {
+          return { ...list, quantity: quantity + value };
+        }
+        return list;
+      })
+    );
+
+    changeQuantity(quantity + value, cartId);
+  };
+
+  const changeQuantity = (quantity, cartId) => {
+    fetch('http://10.58.52.228:8002/carts', {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+        Authorization: localStorage.getItem('TOKEN'),
+      },
+      body: JSON.stringify({
+        cartId: cartId,
+        quantity: quantity,
+      }),
+    })
+      .then(response => response.json())
+      .then(data => {
+        if (data.message === 'error') {
+          alert('다시 시도해주세요');
+        }
+        setCartData(data.update);
+      });
+  };
+
+  const deleteCartList = cartId => {
+    fetch(`http://10.58.52.228:8002/carts?cart_id=${cartId}`, {
+      method: 'DELETE',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+        Authorization: localStorage.getItem('TOKEN'),
+      },
+    })
+      .then(response => response.json())
+      .then(data => {
+        if (data.deleteItem) {
+          alert('삭제 되었습니다.');
+          setCartData(data.deleteItem);
+        }
+      });
+  };
+
   return (
-    <div className="cart">
-      <div className="cartProducts">
-        <div className="cartProductsHeader">
-          <div>상품명</div>
-          <div>용량</div>
-          <div>수량</div>
-          <button
-            className="closeButton"
-            onClick={() => {
-              setIsCartOpen(false);
-            }}
-          >
-            X
-          </button>
+    <div className="cartModal">
+      <div className="cart">
```

Comment:
```suggestion
    <div className="cart">
      <div className="cartContainer">
```
이런식으로 className이 변경되어야 할 것 같네요!